res/controllers/Korg-KAOSS-DJ-scripts.js

```diff
@@ -0,0 +1,194 @@
+// Korg KAOSS DJ controller mapping for Mixxx
+// Seb Dooris, Fayaaz Ahmed, Lee Arromba
+
+function KAOSSDJ() {};
```

Comment:
Please change this to `var KAOSSDJ = {};`. Declaring the variable as an empty function like that confuses people who are new to JS.